src/languages/es.js

```diff
@@ -346,6 +346,7 @@ export default {
         sendMoney: 'Enviar dinero',
         pay: 'Pagar',
         viewDetails: 'Ver detalles',
+        pending: 'Pendiente',
```

Comment:
@rocio do you mind checking these es translations?

Reply:
I think there is this SO for the translations https://stackoverflowteams.com/c/expensify/questions/10316/10317#10317